### PR TITLE
New component recaptcha

### PR DESCRIPTION
**Pull request by Ian-c-cameron**
### Please check if the PR fulfills these requirements:

- [ ] After these changes, the app was run and still works as expected
- [ ] Tests for these changes were added (if applicable)
- [ ] All existing unit tests were run and still pass

### Please specify the type of change this PR introduces (Bug fix, feature addition, content update, chore, etc.):
New Component: Recaptcha
Integrates Google Recaptcha and validates using recaptcha-service

### CODE REVIEW

File: src/components/Recaptcha.vue [hunk 1]
```diff
@@ -0,0 +1,167 @@
+<template>
+  <div>
+    <vue-recaptcha :sitekey="this.publicKey"
```
Comment (harrymaynard-maximus) [inline src/components/Recaptcha.vue:3]: You shouldn't need to specify the `this` context here. By default Vue uses `this` as the context, so you set this prop via: `:sitekey="publicKey"`.

**Review by harrymaynard-maximus — CHANGES_REQUESTED** [review]
One minor change requested please.

**Comment by harrymaynard-maximus** [comment]
Also, question: does this version of library (which includes the "vue-recaptcha" package) work in OOP without adding "vue-recaptcha" as a dependency?

**Comment by harrymaynard-maximus** [comment]
Can you also add a unit testing suite? As a minimum, we want to confirm that the component renders.

File: src/components/Recaptcha.vue (hunk 1 above)
Comment (Ian-c-cameron) [reply src/components/Recaptcha.vue]: I have made the change.

**Comment by Ian-c-cameron** [comment]
> Also, question: does this version of library (which includes the "vue-recaptcha" package) work in OOP without adding "vue-recaptcha" as a dependency?

I have tested linked to OOP, and it builds without any changes.  I do not forsee any difference once the new version is published.
